bin/spark-evaluate.js

```diff
@@ -47,7 +47,7 @@ const ieContract = new ethers.Contract(
 )
 const ieContractWithSigner = ieContract.connect(signer)
 const web3Storage = new Web3Storage({ token: WEB3_STORAGE_API_TOKEN })
-const fetchMeasurements = (cid) => fetchMeasurementsViaClient(web3Storage, cid)
+const fetchMeasurements = (cid) => fetchMeasurementsTrustless(web3Storage, cid)
```

Comment:
This looks like a bug to me, fetchMeasurementsTrustless accepts a single argument:

```js
export const fetchMeasurementsTrustless = async cid => {
```

Can you please add some tests for this function?

Reply:
Just spotted it as well. How do you propose we test this? This is the binary entry point 🤔

Reply:
> How do you propose we test this? This is the binary entry point 🤔

Ah, you are right. We haven't figured out how to automatically test our binary entry points yet, and I guess we have bigger fish to fry.

How about testing this manually - run the service locally (via npm start?), wait for the first measurement to be committed on the chain, and check it's correctly fetched.

Reply:
I guess Typescript is the most exciting option for bin script validation. Will run manually though for now